main.rb

```diff
@@ -2,7 +2,7 @@
 
 def main
   app = App.new
-  app.display_menu
+  app.run
 end
```

Comment:
- [ ] Great job working on the functionality for the refactor but here as mentioned in the requirements Make sure that the role of `main.rb` is only to start your code and ask the user for the initial choice of options (or to quit). So I highly recommend adding the functionality for them here I appreciate that you added them in many file but let's make as the required 🔥✅♻

![image](https://github.com/mwismann/oop_school_library/assets/58553711/92586369-d939-49c8-bc2f-12977a38f842)